engine/openbor.c

```diff
@@ -573,6 +573,7 @@ int					nosave				= 0;
 int                 nopause             = 0;                    // OX. If set to 1 , pausing the game will be disabled.
 int                 noscreenshot        = 0;                    // OX. If set to 1 , taking screenshots is disabled.
 int                 endgame             = 0;
+int                 allow_cheats        = -1;                   // Kratus (04-2022) Now the "nocheats" function can be changed by script using the openborvariant "cheats"
```

Comment:
I'm not sure it's necessary to add another variable here. We can probably add the functionality by making an existing cheat variable bitwise and creating a set of masks (same principal as in screen series). I'll take a closer look at how cheats work.

Reply:
Do you mean a new openborvariant or a new variable inside the openbor.c?
In this case I'm using the already existing openborvariant("cheats") to avoid creating a new one.

On the other hand, it would be good if we could turn on/off every cheat option using a bitwise logic with a constan's list as reference.

Reply:
I didn't know the variable was already there. I thought you added it. Sorry about that. In any case, yes, all the cheats can and should be handled with a single bitwise integer (assuming the cheat options are all true/false and there are not more than 31 of of them).